testing/traefik/traefik.toml

```diff
@@ -1,38 +1,27 @@
-debug = false
+[global]
+  checkNewVersion = true
+  sendAnonymousUsage = true
 
-defaultEntryPoints = ["http", "https"]
+[log]
+  level = "INFO"
 
 [entryPoints]
   [entryPoints.http]
   address = ":80"
-    [entryPoints.http.redirect]
-    entryPoint = "https"
```

Comment:
There's no direct replacement but https://github.com/containous/traefik/issues/4863#issuecomment-534065126 works for me